site/app/templates/polls/PollForm.twig

```diff
@@ -140,20 +140,33 @@
             <option value="when_ended" {{ poll is not null and poll.getReleaseAnswer() == 'when_ended' ? "selected" : "" }}>After Poll Ended</option>
             <option value="always" {{ poll is not null and poll.getReleaseAnswer() == 'always' ? "selected" : "" }}>Always</option>
         </select>
-        <br/>
+        <br><br>
+        <div>
+            <label for = "poll-custom-options" class="option-title credit-form-item">
+                Allow student-written responses?
+            </label>
+            <input id ="poll-custom-options" class="credit-box credit-form-item" type="checkbox" name="poll-custom-options" {{poll is not null and poll.getAllowsCustomResponses() ? "checked" : ""}}>
+            <i>
+                <br />
+                Are you certain you want to allow student-written responses? Students will be able to submit responses that will be visible to other students.<br />
+                Responses may become cluttered. If the question-type is not a survey, these responses will be labeled incorrect by default.
```

Comment:
Do we really need to enforce a line break between these two sentences, or should we rely on the default text-wrapping behavior?

Reply:
Removed in [83f0600](https://github.com/Submitty/Submitty/pull/10197/commits/83f060067c93889d23d498e858b7c4d9b2c0abef)